projects/v2/src/app/modules/search/search.component.ts

```diff
@@ -102,7 +102,13 @@ export class SearchComponent implements OnInit, AfterViewInit {
     this.structures = [...searchSet];
     this.filteredstructuresMulti.next(this.structures.slice());
 
-    this.ga.eventEmitter('nav_search_clicked', GaCategory.NAVBAR, 'Click Search Box', GaAction.CLICK);
+    // Show search dropdown
+    this.store.dispatch(new OpenSearch());
+    this.searchFieldContent.nativeElement.focus();
```

Comment:
In this line, I'm trying to make the search input auto focus after it pops up, but no matter what I try, I can't get it to work. Here are a few approaches I've tried:

- `nativeElement.focus()` (what I have now): https://stackoverflow.com/questions/49634477/on-click-set-focus-on-textarea-angular-5/49636692
- `cdkFocusInitial `(didn't work): https://stackoverflow.com/questions/52143052/how-to-manually-set-focus-on-mat-form-field-in-angular-6
- One of the earlier issues I was having was with ViewChild returning undefined for the input element (as it was hidden with ngIf). I fixed this using a setter as shown here: https://stackoverflow.com/questions/39366981/viewchild-in-ngif but it ultimately didn't focus the element either. 
- Finally, I also tried using angular material's interface, `MatInput.focus()`, instead of `nativeElement.focus()` but that didn't work.

Reply:
Cool that's alright!